app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -112,6 +112,15 @@ def vm_param_groups(template)
           :constraints   => [
             OrchestrationTemplate::OrchestrationParameterBoolean.new
           ]
+        ), OrchestrationTemplate::OrchestrationParameter.new(
+          :name          => param_name('guest_customization', [vm_idx]),
+          :label         => 'Guest customization',
+          :description   => 'Allow guest customization',
+          :data_type     => 'boolean',
+          :default_value => vm.guest_customization,
+          :constraints   => [
+            OrchestrationTemplate::OrchestrationParameterBoolean.new
+          ]
```

Comment:
May I suggest we put this new checkbox above `admin_password` field? I think it makes more sense there, since admin password will NOT be set if we uncheck this box.